pkg/storage/sqlite/sqlite.go

```diff
@@ -1033,7 +1033,7 @@ func (s *Datastore) ReadChanges(ctx context.Context, store string, filter storag
 
 // IsReady see [sqlcommon.IsReady].
 func (s *Datastore) IsReady(ctx context.Context) (storage.ReadinessStatus, error) {
-	return sqlcommon.IsReady(ctx, s.db)
+	return sqlcommon.IsReady(ctx, s.db, "sqlite")
```

Comment:
Is it "sqlite" or "sqlite3"? https://github.com/pressly/goose/blob/9c748e24f4b2cb09afa0dd252ed3548c9f17d51b/internal/dialect/dialects.go#L4

Reply:
In our tests, we were using sqlite, that’s why i decided to keepthis way -- But, I can change that if req.

Reply:
Both are OK. https://github.com/pressly/goose/blob/v3.24.1/dialect.go#L40